Makefile

```diff
@@ -26,10 +25,13 @@ release-deps:
 release: release-deps
 	gox -os $(OS) -arch $(ARCH) -ldflags "-X main.Version=$(VERSION)" -output ".releases/{{.OS}}/{{.Arch}}/{{.Dir}}" ./cmd/carbon-relay-ng
 
+check-formating:
+	@bash -c "diff -u <(echo -n) <(gofmt -d ./)"
```

Comment:
why not using make fmt command ?

Reply:
the make fmt formats, this just checks and returns 1 if there are formatting error. Like this it won't pass the tests if the formatting is incorrect (for whatever reason)